test/__snapshots__/fixtures.test.ts.snap

```diff
@@ -169,6 +169,118 @@ import React from 'react'
 `;
 
 exports[`fixtures lint code blocks should work as expected: code-blocks.mdx 1`] = `
+[
+  {
+    "column": 7,
+    "endColumn": 49,
+    "endLine": 4,
+    "fix": {
+      "range": [
+        117,
+        193,
+      ],
+      "text": "const deployProjectCommand: DeployProjectCommand = new DeployProjectCommand();",
+    },
+    "line": 4,
+    "message": "'deployProjectCommand' is never reassigned. Use 'const' instead.",
+    "messageId": "useConst",
+    "nodeType": "Identifier",
+    "ruleId": "prefer-const",
+    "severity": 2,
+  },
+  {
+    "column": 1,
+    "endColumn": 10,
+    "endLine": 20,
+    "fix": {
+      "range": [
+        0,
+        685,
+      ],
+      "text": "\`\`\`ts
+import { CommandRegister } from "@foo";
+export function registerCommands(commandRegister: CommandRegister) {
+  let deployProjectCommand: DeployProjectCommand = new DeployProjectCommand();
+  commandRegister.addCommand(deployProjectCommand);
+}
+\`\`\`
+
+{/* eslint-skip */}
+
+\`\`\`ts
+import { CommandRegister } from "@foo";
+export function registerCommands(commandRegister: CommandRegister) {
+  let deployProjectCommand: DeployProjectCommand = new DeployProjectCommand();
+  commandRegister.addCommand(deployProjectCommand);
+}
+\`\`\`
+
+# Heading
+
+# Heading
+
+# Heading {/* eslint-disable-line mdx/remark */}
+
+{/* eslint-disable-next-line mdx/remark */}
+
```

Comment:
https://github.com/remarkjs/remark-lint/blob/main/packages/remark-lint-no-missing-blank-lines/readme.md related, should it takes comments into account? @wooorm